src/ui/components/Rating/index.js

```diff
@@ -140,7 +140,7 @@ export class RatingBase extends React.Component<InternalProps, StateType> {
         key: `rating-${thisRating}`,
         onClick: undefined,
         onMouseEnter: () => this.onHoverStar(thisRating),
-        title: this.renderTitle(rating, readOnly, thisRating),
+        title: this.renderTitle(rating, readOnly || false, thisRating),
```

Comment:
Why not just define `readOnly` in `renderTitle` as `readOnly?`? This `|| false` looks odd to me.

Reply:
I didn't want it to be `readOnly?` in `renderTitle` because that function relies on three positional arguments and I wanted Flow to catch accidental omissions. I think I can solve this a different way, though.